packages/flutter_tools/test/general.shard/screenshot_command_test.dart

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_tools/src/base/common.dart';
+import 'package:flutter_tools/src/commands/screenshot.dart';
+import 'package:matcher/matcher.dart';
+
+import '../src/common.dart';
+import '../src/context.dart';
+
+void main() {
+  final Matcher expectToolError = throwsA(const TypeMatcher<ToolExit>());
```

Comment:
There should already be a matched called throwsToolExit

Reply:
Switched to using this.